lib/swift/Sources/TProtocol.swift

```diff
@@ -132,12 +137,13 @@ public extension TProtocol {
   func skip(type: TType) throws {
     switch type {
     case .bool:   _ = try read() as Bool
-    case .i8:   _ = try read() as UInt8
+    case .i8:   _ = try read() as Int8
```

Comment:
was this a bug?

Reply:
yeah I'm not sure if the spec changed at some point, but the docs now define `i8` as an "An 8-bit signed integer" https://thrift.apache.org/docs/types so I think it should be Int8

Reply:
The `byte` type was **always** signed, like all Thrift integer types. So we renamed the type specifier a while ago from byte to `i8` mostly to remove that constant source of confusion - a byte is considered unsigned in many languages. Nevertheless, [for compatibility reasons](https://www.mail-archive.com/dev@thrift.apache.org/msg50144.html) you still can use `byte` in the IDL.